css/style.css

```diff
@@ -0,0 +1,167 @@
+* {
+	font-family: "raleway";
+	color: #f00;
+}
+/*Header definition*/
+.header {
+	margin: 0;
+	padding: 0;
+	width: 100%;
+	display: flex;
+	flex-wrap: wrap;
+	display: -webkit-flex;
+	-webkit-flex-wrap: wrap;
+}
+/*Responsive Logo*/
+.img-responsive {
+	max-width: 100%;
+	margin-top: 20px;
+}
+/*Header Line*/
+hr {
+	height: 3px;
+	background-color: #a9a9a9;
+	margin-bottom: 28px;
+}
+/*Dividing line*/
+.figure-line {
+	width: 100%;
+	margin: 0;
+}
+/*Main center image*/
+.Image-Technology {
+	width: 100%;
+	margin: 0;
+}
+/*images*/
+img {
+	max-width: 100%;
+	height: auto;
+}
+/*Links Github*/
+a {
+	color: #f00;
+	font-size: 11px;
+}
+/*Text alignment and width format*/
+.header-text {
+	width: 90%;
+	text-align: right;
+}
+/*Size, color and style of the first title*/
+.header-title {
+	color: #5e6164;
+	font-size: 50px;
+	font-weight: lighter;
+	margin-bottom: 5px;
+}
+/*Color and margin subtitle*/
+.header-subtitle {
+	color: #5e6164;
+	margin-top: 5px;
+}
+/*Featured work*/
+.featured-work {
+	margin: 0;
+	padding: 0;
+	width: 100%;
+	display: flex;
+	flex-wrap: wrap;
+	display: -webkit-flex;
+	-webkit-flex-wrap: wrap;
+}
+/*Featured work*/
+.featured-title {
+	width: 100%;
+	color: #b6b7b3;
+	margin-bottom: 10px;
+	font-size: 30px;
+}
+/*Size Images and Text*/
+.Size-Images {
+	width: 32.5%;
+	text-align: center;
+}
+/*Subtitle properties font-size and color*/
+.Subtitle {
+	color: #006400;
+	font-size: 15px;
+}
+/*Styles of links*/
+.link-text {
+	color: #191970;
+	font-size: 13px;
+}
+/*Container definition*/
+.container {
+	display: flex;
+	flex-wrap: wrap;
+	display: -webkit-flex;
+	-webkit-flex-wrap: wrap;
+	width: 100%;
+}
+/*Properties over 800px*/
+@media screen and (min-width: 800px) {
+	.container {
+		width: 800px;
+		margin: auto;
+	}
+	.Size-Images {
+		margin-right: 10px;
+	}
+	.Size-Images:last-of-type {
+		margin-right: auto;
+	}
+}
+/*Properties over 550px to 800px*/
+@media screen and (min-width: 551px) and (max-width: 800px) {
+	.header {
+		margin: 0 10px 0 10px;
+	}
+	.figure-line {
+		margin: 0 10px 0 10px;
+	}
+	.Image-Technology {
+		margin: 0 10px 0 10px;
+	}
+	.featured-work {
+		margin: 0 10px 0 10px;
+	}
+	.Size-Images {
+		margin-right: 10px;
+		width: 32%;
+	}
+	.Size-Images:last-of-type {
+		margin-right: 0;
+	}
+	.Subtitle {
+	color: #5f6050;
+	font-size: 25px;
+	margin: 7px 0 7px 0;
+	}
+}
+/*Properties for under 551px*/
+@media screen and (max-width: 550px) {
+	.header-text {
+		width: 100%;
+		text-align: center;
+	}
+	.featured-title {
+		text-align: center;
+	}
+	.Size-Images {
+		width: 100%;
+	}
+	.Logo-Responsive {
+		width: 100%;
+		text-align: center;
+	}
+	.img-responsive {
+		max-width: 40%;
+		display: inline-block;
+		vertical-align: middle
+	}
```

Comment:
#### Awesome :white_check_mark: 

Your code looks very clean and easy to read, thanks!. Using indentations, documentation and style consistency, really makes a huge difference in the code readability.